src/styles/shared/_interaction-state.scss

```diff
@@ -24,3 +24,16 @@
   }
   background-image: $backgrounds;
 }
+
+@mixin list-single-selection-state {
```

Comment:
Not sure how I feel about this mixin name. Might I suggest some variation of:
- `action-list-indicator`
- `list-pseudo-indicator`
- `list-selected-indicator`